File: Fundamentals/Functions - Lab/calculations.py
def calculation(operator, first_number, second_number):
    if operator == 'multiply':
        return first_number * second_number
    elif operator == 'divide':
        return int(first_number/second_number)
    elif operator == 'add':
        return first_number + second_number
    elif operator == 'subtract':
        return first_number - second_number


operator = input()
first_number = int(input())
second_number = int(input())
print(calculation(operator=operator, first_number=first_number, second_number=second_number))

File: JavaFundamentals/src/Methods/Lab/Calculations.java
package Methods.Lab;

import java.util.Scanner;

public class Calculations {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String action = scanner.nextLine();
        int firstNumber = Integer.parseInt(scanner.nextLine());
        int secondNumber = Integer.parseInt(scanner.nextLine());

        switch (action) {
            case "add" -> add(firstNumber, secondNumber);
            case "subtract" -> subtract(firstNumber, secondNumber);
            case "multiply" -> multiply(firstNumber, secondNumber);
            case "divide" -> divide(firstNumber, secondNumber);
        }
    }

    private static void add(int a, int b) {
        System.out.println(a + b);
    }

    private static void subtract(int a, int b) {
        System.out.println(a - b);
    }

    private static void multiply(int a, int b) {
        System.out.println(a * b);
    }

    private static void divide(int a, int b) {
        System.out.println(a / b);
    }
}
